modules/rbac-creating-cluster-admin.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * authentication/using-rbac.adoc
+
+ifdef::openshift-enterprise,openshift-origin[]
```

Comment:
Why is this line here? There's not a matching ending one. And if it is required, I'd think we'd want to put the conditional in the assembly? wdyt?

Reply:
It was an oversight from the base file that I used. It seems that conditionals are split between modules and assemblies, and one of the modules I started this from included it.

I've removed the conditional from the module. The other sections on defining user- and cluster- roles don't have a similar conditional, so I believe it's safe to keep out of the assembly.